### ISSUE TITLE
Install failed on Fedora 39

### ISSUE THREAD

**Issue by voidastro4** [pwsafe/pwsafe#1053]
F38 package fails to install on F39

```
 Problem: conflicting requests
  - nothing provides libwx_baseu-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_baseu-3.0.so.0(WXU_3.0)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_baseu_net-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_adv-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_adv-3.0.so.0(WXU_3.0)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_aui-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_aui-3.0.so.0(WXU_3.0)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_core-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_core-3.0.so.0(WXU_3.0)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_core-3.0.so.0(WXU_3.0.5)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_html-3.0.so.0()(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides libwx_gtk3u_html-3.0.so.0(WXU_3.0)(64bit) needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides wxBase3 needed by passwordsafe-1.18-1.x86_64 from @commandline
  - nothing provides wxGTK3 needed by passwordsafe-1.18-1.x86_64 from @commandline
```

wxBase3-devel wxGTK3-devel

was renamed to

wxBase-devel wxGTK-devel

I was able to build pwsafe on F39 and the binary is working. Building RPM failed.

**Comment by ronys** [comment]
Fixed the build in commit 909efa1 
The RPM build failure seems unrelated to pwsafe.

**Comment by ronys** [comment]
Worked around cpack issue. RPM may be found here: https://github.com/pwsafe/pwsafe/releases/download/1.18.0/passwordsafe-fedora39-1.18.rpm
